letsdrill.rb

```diff
@@ -1,24 +1,40 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+  if integer > 89
+    letter_grade = "A"
+  elsif integer > 79
+    letter_grade = "B"
+  elsif integer > 69
+    letter_grade = "C"
+  elsif integer > 59
+    letter_grade = "D"
+  else
+    letter_grade = "F"
```

Comment:
The whole `if/elsif/else` construct is an expression that returns whichever inner expression that is evaluated.

This should work the same:

```ruby
def get_letter_grade(integer)
  letter_grade = if integer > 89
                   "A"
                   ...
                 end

  letter_grade
end
```
Likewise, the local variable isn't needed:

```ruby
def get_letter_grade(integer)
  if integer > 89
    "A"
    ...
  end
end
```